server/views/tasks/index.njk

```diff
@@ -26,8 +26,7 @@
             </div>
           </div>
           <div class="govuk-grid-row">
-            <div class="govuk-grid-column-one-third">
-              {{ govukSelect({
+            <div class="govuk-grid-column-one-third">              {{ govukSelect({
```

Comment:
could you revert this formatting so that govukSelect is on a new line?

Reply:
👍